src/Robo/Commands/Doctor/ComposerCheck.php

```diff
@@ -104,16 +91,6 @@ protected function setComposerLock() {
     return [];
   }
 
-  /**
-   * Get composer.lock.
-   *
-   * @return array
-   *   Array.
-   */
-  public function getComposerLock() {
```

Comment:
I see we're not using this anywhere in our codebase. Could someone else use it from an external source?

Reply:
Yeaahhhh.... I guess it's _theoretically_ possible. Just to be safe, I'll put it back.